src/main/java/seedu/address/model/person/Person.java

```diff
@@ -34,6 +37,7 @@ public Person(Name name, ClassGroup classGroup, Email email, Optional<Phone> pho
         this.phone = phone;
         this.telegram = telegram;
         this.github = github;
+        this.attendance = new Attendance();
```

Comment:
This reminds me that we have to change the update Command to copy over the attendance and notes.

Reply:
That is correct, I can create a issue for it so that we do not forget